softfloat/specialize.h

```diff
@@ -295,6 +298,9 @@ INLINE
 struct uint128 softfloat_commonNaNToF128UI( const struct commonNaN *aPtr )
 {
     struct uint128 uiZ;
+
+    (void) aPtr;
```

Comment:
@scottj97 Note I didn't change this one, because unnamed parameters are a C++ feature.  (They are coming to C2x, but they aren't standard yet.)